control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3678,6 +3680,28 @@ func (r *HostedControlPlaneReconciler) reconcileAutoscaler(ctx context.Context,
 		return fmt.Errorf("availability prober image not found")
 	}
 
+	// Check if PodSafeToEvict annotation exists in hcp pods
+	hcpPods := &corev1.PodList{}
```

Comment:
I prefer if we consciously set this annotation in the pods we specifically know they are preventing draining because of an empty dir.
So we need to make this part of each specific component reconciliation.

For that you can have a look at how we use the support/config package and SetDefaults or SetRestartAnnotation. You don't need to use exactly that as that package needs refactor to provide us valuable abstractions.

Reply:
Makes sense. To address this, I've follow 2 ways:
- Annotate the ETCD StatefulSet
- Create a new [PR to the cluster-network-operator](https://github.com/openshift/cluster-network-operator/pull/1822) which include this annotation in the `ovnkube-master` pods